tensorflow/python/ops/metrics_impl.py

```diff
@@ -2049,22 +2024,98 @@ def recall(labels,
         updates_collections=None,
         name=None)
 
-    def compute_recall(true_p, false_n, name):
-      return array_ops.where(
-          math_ops.greater(true_p + false_n, 0),
-          math_ops.div(true_p, true_p + false_n), 0, name)
+    return _div_metric(true_p, true_positives_update_op, true_p + false_n,
+                       true_positives_update_op + false_negatives_update_op,
+                       metrics_collections, updates_collections, 'value')
 
-    rec = compute_recall(true_p, false_n, 'value')
-    update_op = compute_recall(true_positives_update_op,
-                               false_negatives_update_op, 'update_op')
+@tf_export('metrics.sensitivity')
+def sensitivity(labels,
+                predictions,
+                weights=None,
+                metrics_collections=None,
+                updates_collections=None,
+                name=None):
+  """Same as `tensorflow.metrics.recall`"""
+  return recall(labels, predictions, weights, metrics_collections,
+                updates_collections, name)
```

Comment:
Instead of making this function an alias, use
name or "sensitivity"
instead of
name

That way the name reflects the function the user called.

Reply:
good point. agreed

Reply:
@josh11b  Do you mean something like below?
```python
def sensitiviey(...., name=None):
  if name is None:
    name = "sensitivity"

  return recall(...., name)
```